### PR TITLE
Kampala: Zer0

### PR DESCRIPTION
**Pull request by nicohasa**

### CODE REVIEW

File: test/clj/game_test/cards/hardware.clj [hunk 1]
```diff
@@ -1177,3 +1177,19 @@
...
+  ;; Zer0 - Once per turn, deal 1 damage to self, to gain 1 credit and 2 cards.
+  (do-game
+    (new-game (default-corp)
+              (default-runner [(qty "Zer0" 1) (qty "Sure Gamble" 2) (qty "Corroder" 1)]))
+    (starting-hand state :runner ["Zer0" "Corroder"])
+    (take-credits state :corp)
+    (play-from-hand state :runner "Zer0")
+    (is (= 4 (:credit (get-runner))) "Runner has 4 credits")
+    (let  [z (get-hardware state 0)]
+      (print z)
```
Comment (Saintis) [inline test/clj/game_test/cards/hardware.clj]: Normally we don't put print functions in the tests

File: test/clj/game_test/cards/hardware.clj [hunk 2]
```diff
@@ -1177,3 +1177,19 @@
    (prompt-choice :runner "Expose a card")
    (prompt-select :runner (get-ice state :archives 0))
    (is (= 3 (:credit (get-runner))) "Gained 1 more credit from exposing")))
+
+(deftest zer0
+  ;; Zer0 - Once per turn, deal 1 damage to self, to gain 1 credit and 2 cards.
+  (do-game
+    (new-game (default-corp)
+              (default-runner [(qty "Zer0" 1) (qty "Sure Gamble" 2) (qty "Corroder" 1)]))
```
Comment (Saintis) [inline test/clj/game_test/cards/hardware.clj]: The single `qty` cards could be replaced with just a string of the card name (e.g. `"Zer0"`)

File: test/clj/game_test/cards/hardware.clj (hunk 1 above)
Comment (nicohasa) [reply test/clj/game_test/cards/hardware.clj]: wups, that's left after I tried to figure out what went wrong at some point

**Comment by nealterrell** [comment]
We may need to look at making runner-install use `pay-sync`, but for now this should be OK.

**Pull request merged**